packages/Interface/test/tstC_API.c

```diff
@@ -0,0 +1,81 @@
+#include <DTK_C_API.h>
```

Comment:
How would you combine them in a single file? You need both "C" and "C++" parts compiled simultaneously but using C and C++ compilers independently.